CHANGELOG.next.toml

```diff
@@ -261,3 +261,9 @@ For more information, see the [Change Log Discussion](https://github.com/awslabs
 meta = { "breaking" = true, "tada" = false, "bug" = false }
 references = ["smithy-rs#3014"]
 author = "rcoh"
+
+[[aws-sdk-rust]]
+message = "`ProviderConfig` will now respect both `use_fips` and `use_dual_stack` when those settings are configured in a user's environment or profile."
+references = ["aws-sdk-rust#882", "smithy-rs#3007"]
+meta = { "breaking" = false, "tada" = true, "bug" = true }
+author = "Velfi"
```

Comment:
message should be updated to be customer facing (referencing SSO/STS credential providers)